hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1062,6 +1065,14 @@
   </description>
 </property>
 
+<property>
+  <name>fs.viewfs.overload.scheme.target.gs.impl</name>
+  <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS</value>
+  <description>The HttpFileSystem for view file system overload scheme
```

Comment:
nit: change "HttpFileSystem" to "Google Cloud Storage file system"

Reply:
@jojochuang done.